compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -1676,16 +1676,11 @@ private void defineTypeInMainScope(BTypeSymbol typeDefSymbol, BLangTypeDefinitio
         }
     }
 
-    private BType defineSymbolsForCyclicTypeDefinitions(BLangTypeDefinition typeDef, SymbolEnv env) {
+    private BType createSymbolAndDefineInScope(BLangTypeDefinition typeDef, SymbolEnv env) {
```

Comment:
Shall we rename the two methods? Maybe `defineSymbolForCyclicTypeDefinition` and `getCyclicType`. wdyt?